docs/_documentations/mdt-eclipse-buildproject.md

```diff
@@ -10,9 +10,9 @@ order: 40
 parent: mdteclipseoverview
 ---
 
-# Building Codewind projects
+# Codewind build logs
 
-When auto build is enabled for a project, Codewind for Eclipse detects when you make a change and starts a build automatically. If you have disabled auto build for the project, you can start a build manually when you have made a change or a set of changes:
+If you enable auto build for a project, Codewind for Eclipse, Eclipse Che, and Visual Studio Code (VS Code) detects when you make a change and starts a build automatically. If you have disabled auto build for a project, you can start a build manually when you have made a change or a set of changes:
 
-1. Right-click your project in the **Codewind Explorer** view and select **Build**.
-2. Wait for the project state to return to **Running** or **Debugging** in the **Codewind Explorer** view and then test your changes.
+1. Right-click your project in the **Codewind Explorer** view > select **Build**.
```

Comment:
Keep the "and" here instead of the `>`. The `>` is OK for if you have a long string of things to click, such as, "Go to `Program>Files>Product_Page` and click the file you want to edit."